tests/test_cli.py

```diff
@@ -2,14 +2,21 @@
 import tempfile
 from unittest.mock import patch
 
+import pytest
+
 from markdown_it.cli import parse
 
 
 def test_parse():
     with tempfile.TemporaryDirectory() as tempdir:
         path = pathlib.Path(tempdir).joinpath("test.md")
         path.write_text("a b c")
-        assert parse.main([str(path)])
+        assert parse.main([str(path)]) == 0
+
+
+def test_parse_fail():
+    with pytest.raises(SystemExit):
+        parse.main(["/tmp/nonexistant_path/for_cli_test.md"])
```

Comment:
SystemExit per se doesn't mean that exit code is non-zero. We can assert that with something like:
```suggestion
    with pytest.raises(SystemExit) as exc_info:
        parse.main(["/tmp/nonexistant_path/for_cli_test.md"])
    assert exc_info.value.code == 1
```